src/vulkan.cpp

```diff
@@ -1025,30 +1026,49 @@ void render_imgui(swapchain_stats& data, struct overlay_params& params, ImVec2&
             gpu_text = params.gpu_text.c_str();
          ImGui::TextColored(data.colors.gpu, "%s", gpu_text);
          ImGui::TableNextCell();
-         right_aligned_text(ralign_width, "%i", gpu_info.load);
-         ImGui::SameLine(0, 1.0f);
-         ImGui::Text("%%");
+         if(params.enabled[OVERLAY_PARAM_ENABLED_gpu_load_change]){
+            if(gpu_info.load >= params.gpu_load_high){
```

Comment:
Might as well get just color from if statement, less duplicate code of right_aligned_text etc.